### PR TITLE
proc_loadvg: fixes

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: src/proc_loadavg.c [hunk 1]
```diff
@@ -211,21 +211,24 @@ int proc_loadavg_read(char *buf, size_t size, off_t offset,
...
 			/*
 			 * In locate_node() above, pthread_rwlock_unlock() isn't used
 			 * because delete is not allowed before read has ended.
 			 */
 			pthread_rwlock_unlock(&load_hash[hash].rdlock);
-			return 0;
+			lxcfs_error("AAAA");
+			return read_file_fuse("/proc/loadavg", buf, size, d);
+		} else {
+			lxcfs_error("BBBB");
```
Comment (stgraber) [inline src/proc_loadavg.c]: leftover debugging

Comment (brauner) [reply src/proc_loadavg.c]: fixed

**Comment by brauner** [comment]
Travis can't be reached somehow.

**Comment by stgraber** [comment]
seems back now

**Pull request merged**